### ISSUE TITLE
[REQUEST] Font settings

### ISSUE THREAD

**Issue by milen-prg** [jczic/ESP32-MPY-Jama#60]
Please, can add Font settings for the code editor and the user interface also, because on small display (laptop) all is too microscopic.

**Comment by BOB63** [comment]
I fully agree with @milen-prg , the font used are too small and should be appreciated the possibility to change font.

**Comment by jczic** [comment]
Hi and thank you @milen-prg and @BOB63 ♥

I'm sorry, but unfortunately I'm not working on it at the moment :(
I'm co-founding an AI start-up dedicated to e-commerce customer service... (www.achille.ai).

However, you can always open the package/zip and go to the CSS file to try and change the fonts and sizes.
It's Web format behind the UX ;)
